content/en/logs/log_collection/go.md

```diff
@@ -78,9 +78,13 @@ func main() {
 }
 ```
 
-**Connect Logs and Traces**
+## Connect your service across Logs and APM
```

Comment:
I believe we should keep the word `traces`.

We are not saying Log Management here as we are talking about the data points, not the products. So we are connecting the traces and the logs, not Log Management and APM.

Reply:
Valid point, will update the headers